handbook/engineering/distribution/recurring_processes.md

```diff
@@ -118,33 +118,82 @@ come prepared to talk about the following:
 
 ## Weekly
 
-### Internal sync
+### Company meeting
```

Comment:
It feels redundant/duplicative to me to document this here. I think it would be weird if each team documented that they attend company meeting.

Reply:
I understand it is duplicative, but it is also very minimal.

The main reason I think it is worth calling out here is because in interviews I find it useful often to be able to answer candidates questions for me about "what does a month for me look like in terms of meetings, syncs, etc." with a single link